### DISCUSSION TITLE
Met the same issue like #3906 with v0.11.0 version

### DISCUSSION THREAD

**Discussion by shaozhenged** [milvus-io/milvus#4525] [Q&A and General discussion]
I noticed that the v0.10.4  fixed the issue #3906 which released at 2012-12-03.
I think I met the same issue with v0.11.0 which released at 2010-10-16.

So is there any plans to fix the issue with v0.11.x version?

Thinks

**Comment by yhmo** [comment]
@shaozhenged 
I use same steps try 0.11.x, the #3906 could not reproduce on 0.11.x.
If you meet "Server received critical signal: 11", please tell us your steps.
